### ISSUE TITLE
"Import-Module MarkdownPS" on version 1.5 produces errors on PowerShell Core

### ISSUE THREAD

**Issue by maxim-lobanov** [Sarafian/MarkdownPS#19]
Hello,

## Issue
When I install `MarkdownPS 1.5` module and try to import it, I get the following error:
```
Import-Module : The specified wildcard character pattern is not valid: @{HelpUri=; ScriptBlock=
    [CmdletBinding()]
    [OutputType([string])]
    Param (
...
```
It looks like all functions work as expected after this error but it produces a lot of noise in our logs.

## Possible root cause
Version `1.4` works correctly so it looks like issue comes from `psd1` file that was changed between 1.4 and 1.5 versions (MarkdownPS.psd1 contains a huge inline export in field `FunctionsToExport` in version 1.5)

## Repro steps
1) Use macOS with PowerShell Core
2) Install MarkdownPS
3) Import-Module MarkdownPS

Thank you for implementation and supporting this great PowerShell module, hope this issue will be resolved.

## Full error message
```
VERBOSE: Loading module from path '/Users/***/.local/share/powershell/Modules/MarkdownPS/1.5/MarkdownPS.psd1'.
VERBOSE: Populating RepositorySourceLocation property for module MarkdownPS.
VERBOSE: Loading module from path '/Users/***/.local/share/powershell/Modules/MarkdownPS/1.5/MarkdownPS.psm1'.
Import-Module : The specified wildcard character pattern is not valid: @{HelpUri=; ScriptBlock=
    [CmdletBinding()]
    [OutputType([string])]
    Param (
        [Parameter(
            Mandatory = $false,
            Position = 0,
            ValueFromPipeline = $true,
            ParameterSetName="Source"
        )]
        [ValidateNotNullOrEmpty()]
        [string]$Source,
        [Parameter(
            Mandatory = $false
        )]
        [ValidateNotNullOrEmpty()]
        [string]$Title=$null,
        [Parameter(
            Mandatory = $false
        )]
        [string]$AltText=$null,
        [Parameter(
            Mandatory = $false,
            ParameterSetName="Shields.io"
        )]
        [string]$Subject=$null,
        [Parameter(
            Mandatory = $false,
            ParameterSetName="Shields.io"
        )]
        [string]$Status=$null,
        [Parameter(
            Mandatory = $false,
            ParameterSetName="Shields.io"
        )]
        [ValidateSet("brightgreen","green","yellowgreen","yellow","orange","red","lightgrey","blue")]
        [string]$Color=$null,
        [Parameter(
            Mandatory = $false,
            ParameterSetName="Shields.io"
        )]
        [ValidateSet("plastic","flat","flat-square","social")]
        [string]$Style=$null,
        [Parameter(
            Mandatory = $false
        )]
        [string]$Link=$null
    )
    Begin {
        $output=""
    }
    Process {
        if(-not $Source -and $Color)
        {
            $Subject=$Subject.Replace("-","--").Replace("_","__")
            $Subject=[System.Uri]::EscapeDataString($Subject)
            $Status=$Status.Replace("-","--").Replace("_","__")
            $Status=[System.Uri]::EscapeDataString($Status)
            
            #When running from powershell [System.Uri]::EscapeDataString is not escaping ()
            if(-not $ise)
            {
                $Subject=$Subject.Replace("(","%28").Replace(")","%29")
                $Status=$Status.Replace("(","%28").Replace(")","%29")
            }
            
            $shieldIo="$Subject-$Status-$Color"
            $Source="https://img.shields.io/badge/$shieldIo.svg"
            if($Style)
            {
                $Source+="?style=$Style"
            }
        }
        
        $output+="![$AltText]($Source"
        if($Title)
        {
            $output+=" ""$Title"""
        }
        $output+=")"
        if($Link)
        {
            $output=New-MDLink -Text $output -Link $Link
        }
    }
    End {
        $output
    }
; CmdletBinding=True; DefaultParameterSet=; Definition=
    [CmdletBinding()]
    [OutputType([string])]
    Param (
        [Parameter(
            Mandatory = $false,
            Position = 0,
            ValueFromPipeline = $true,
            ParameterSetName="Source"
        )]
        [ValidateNotNullOrEmpty()]
        [string]$Source,
        [Parameter(
            Mandatory = $false
        )]
        [ValidateNotNullOrEmpty()]
        [string]$Title=$null,
        [Parameter(
            Mandatory = $false
        )]
        [string]$AltText=$null,
        [Parameter(
            Mandatory = $false,
            ParameterSetName="Shields.io"
        )]
        [string]$Subject=$null,
        [Parameter(
            Mandatory = $false,
            ParameterSetName="Shields.io"
        )]
        [string]$Status=$null,
        [Parameter(
            Mandatory = $false,
            ParameterSetName="Shields.io"
        )]
        [ValidateSet("brightgreen","green","yellowgreen","yellow","orange","red","lightgrey","blue")]
        [string]$Color=$null,
        [Parameter(
            Mandatory = $false,
            ParameterSetName="Shields.io"
        )]
        [ValidateSet("plastic","flat","flat-square","social")]
        [string]$Style=$null,
        [Parameter(
            Mandatory = $false
        )]
        [string]$Link=$null
    )
    Begin {
        $output=""
    }
    Process {
        if(-not $Source -and $Color)
        {
            $Subject=$Subject.Replace("-","--").Replace("_","__")
            $Subject=[System.Uri]::EscapeDataString($Subject)
            $Status=$Status.Replace("-","--").Replace("_","__")
            $Status=[System.Uri]::EscapeDataString($Status)
            
            #When running from powershell [System.Uri]::EscapeDataString is not escaping ()
            if(-not $ise)
            {
                $Subject=$Subject.Replace("(","%28").Replace(")","%29")
                $Status=$Status.Replace("(","%28").Replace(")","%29")
            }
            
            $shieldIo="$Subject-$Status-$Color"
            $Source="https://img.shields.io/badge/$shieldIo.svg"
            if($Style)
            {
                $Source+="?style=$Style"
            }
        }
        
        $output+="![$AltText]($Source"
        if($Title)
        {
            $output+=" ""$Title"""
        }
        $output+=")"
        if($Link)
        {
            $output=New-MDLink -Text $output -Link $Link
        }
    }
    End {
        $output
    }
; Options=None; Description=; Verb=New; Noun=MDImage; HelpFile=; OutputType=System.Collections.ObjectModel.ReadOnlyCollection`1[System.Management.Automation.PSTypeName]; CommandType=Function; Source=MarkDownPS; Version=1.4; Visibility=Public; ModuleName=MarkDownPS; Module=MarkDownPS; RemotingCapability=PowerShell; Parameters=System.Collections.Generic.Dictionary`2[System.String,System.Management.Automation.ParameterMetadata]; ParameterSets=System.Collections.ObjectModel.ReadOnlyCollection`1[System.Management.Automation.CommandParameterSetInfo]}
At line:1 char:1
+ Import-Module MarkdownPS -RequiredVersion 1.5 -Verbose
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ CategoryInfo          : NotSpecified: (:) [Import-Module], WildcardPatternException
+ FullyQualifiedErrorId : RuntimeException,Microsoft.PowerShell.Commands.ImportModuleCommand
```

**Comment by Sarafian** [comment]
I published again the 1.6. To share the background story, the manifest file gets generated by a script and somehow the one that was generated and [published](https://www.powershellgallery.com/packages/MarkdownPS/1.5/Content/MarkdownPS.psd1) was completely wrong with powershell code inside. I just updated the version to `1.6` and published again. I tested on my system and it works. I will also remove the `1.5` from the gallery. Thanks for bringing this to my attention.
